packages/form-js-editor/src/features/editor-actions/FormEditorActions.js

```diff
@@ -15,20 +15,40 @@ export default class FormEditorActions extends EditorActions {
   }
 
   _registerDefaultActions(injector) {
-    const commandStack = injector.get('commandStack', false);
+    const commandStack = injector.get('commandStack', false),
+          formFieldRegistry = injector.get('formFieldRegistry', false),
+          selection = injector.get('selection', false);
 
     if (commandStack) {
 
       // @ts-ignore
-      this.register('undo', function() {
+      this.register('undo', () => {
         commandStack.undo();
       });
 
       // @ts-ignore
-      this.register('redo', function() {
+      this.register('redo', () => {
         commandStack.redo();
       });
     }
+
+    if (formFieldRegistry && selection) {
```

Comment:
We have an action called `selectElements` in bpmn-js. Does it make sense to re-use the name? We could throw if more than one element is provided. 

Alternatively use `selectElement` or `selectField`? I'd like to get rid of `formField` from naming on the API surface as all addressable fields are form fields. No need to repeat that specific thing everywhere from my point of view.

Reply:
We could obviously argue that what we call `element` in bpmn-js is called `formField` in form-js. Then using `formField` everywhere is the consistent way of doing things.

Reply:
Let us merge this. We can always unify and clean up later, right?